README.md

```diff
@@ -69,7 +69,7 @@ Report bugs or suggest features using
 
 Account creation
 -------------------------------------------------
-You can get yourself a [free sandbox account](https://www.mangopay.com/get-started/create-sandbox/) or sign up for a [production account](https://www.mangopay.com/get-started/submit-your-app/create-credentials/) (note that validation of your production account can take a few days, so think about doing it in advance of when you actually want to go live).
+You can get yourself a [free sandbox account](https://www.mangopay.com/signup/create-sandbox/) or sign up for a [production account](https://www.mangopay.com/signup/submit-your-app/) (note that validation of your production account can take a few days, so think about doing it in advance of when you actually want to go live).
```

Comment:
Hi @dunglas, thanks for the PR :-) Could you just update the production account link to `https://www.mangopay.com/signup/submit-your-app/go-live/` - ie add the extra `/go-live/` on the end :-)